### PR TITLE
Sin mark character sheet update

### PR DESCRIPTION
**Pull request by diabeatz96**
Updates to have sin-marks work on sheet. Drag and drop works. Rolling sin-marks works and lets you selected based on sin-marks available. Currently dynamically changes based on sin data. Suggested to take my data and drag it into the game, for ease of use.

### CODE REVIEW

File: templates/actor/parts/actor-sin.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="sin-overflow">
+<div class="sin-overflow .sin-page-drop-target">
```
Comment (ahandleman) [inline templates/actor/parts/actor-sin.hbs:1]: think there's an extra . in here

File: templates/actor/parts/actor-sin.hbs [hunk 2]
```diff
@@ -53,4 +62,8 @@
     <button class="clear-sin-marks">Clear Sin Marks</button>
     <button class="delete-sin-mark">Delete Sin Mark</button>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<style>
+
```
Comment (ahandleman) [inline templates/actor/parts/actor-sin.hbs:68]: why the empty style?

**Comment by ahandleman** [comment]
I think fixing the two things I commented on should leave this awesome

**Comment by diabeatz96** [comment]
fixed

**Review by ahandleman — APPROVED** [review]

**Pull request merged**